	Else Solaris gets
libs/icu/icu-src/source/i18n/plurrule.cpp:1884:90:
error: call of overloaded 'pow(int, const int32_t&)' is ambiguous
	per https://tug.org/pipermail/tlbuild/2022q1/005020.html.

--- plurrule.cpp	(revision 61627)
+++ plurrule.cpp	(working copy)
@@ -1881,7 +1881,7 @@
 
 double FixedDecimal::getPluralOperand(PluralOperand operand) const {
     switch(operand) {
-        case PLURAL_OPERAND_N: return (exponent == 0 ? source : source * pow(10, exponent));
+        case PLURAL_OPERAND_N: return (exponent == 0 ? source : source * pow(10.0, exponent));
         case PLURAL_OPERAND_I: return (double) longValue();
         case PLURAL_OPERAND_F: return static_cast<double>(decimalDigits);
         case PLURAL_OPERAND_T: return static_cast<double>(decimalDigitsWithoutTrailingZeros);
